docker/rootfs/usr/local/nginx/conf/nginx.conf

```diff
@@ -173,6 +186,22 @@ http {
             proxy_set_header Host $host;
         }
 
+        location ~ /restream/mp4/(.*)$ {
+            proxy_pass http://go2rtc/api/stream.mp4?src=$1;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection "Upgrade";
+            proxy_set_header Host $host;
+        }
+
+        location /go2rtc/ {
```

Comment:
Looks like `/restream-ws/` and `/go2rtc/` are copies of each other, should be able to consolidate